getting-started/queries.md

```diff
@@ -159,5 +159,63 @@ For more information about these functions, see the API documentation for
 
 </Procedure>
 
-[first]: /api/:currentVersion:/hyperfunctions/first
-[last]: /api/:currentVersion:/hyperfunctions/last
+## Use time buckets to get values
+
+To make it easier to look at numbers over different time ranges, you can use the
+Timescale `time_bucket` function. Time buckets are used to group data, so that
```

Comment:
Maybe add a sentence such as:
_Time buckets represent a specific point in time according to the bucket definition. All timestamps between the time bucket start and the selected length will be truncated down to the bucket timestamp. Time buckets are used to group ..._